src/fetchers/stats-fetcher.js

```diff
@@ -25,11 +25,15 @@ const fetcher = (variables, token) => {
   return request(
     {
       query: `
-      query userInfo($login: String!) {
+      query userInfo($login: String!${
```

Comment:
I added this condition here since GraphQL doesn't take `undefined` and `null` as query parameters.